### DISCUSSION TITLE
Multi-Widget

### DISCUSSION THREAD

**Discussion by E-Stegemann** [Zefau/ioBroker.jarvis#2427] [Ideas / Feature Requests]
Es wäre klasse, wenn es eine Art Multi-Widget geben würde aller "View in Widget" von der Vis Umgebung. Ich habe bspw 5 Kameras die ich über das Image Widget anzeigen lasse mit einer Aktualisierung von 1s. Jetzt wäre es toll wenn man in einem Widget alle 5 Kamera Snapshots abbilden könnte. Entweder per swipe oder eine Art Reiter im Widget selbst.

**Comment by mcuiobroker** [comment]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/module/displayimage/bilder-nebeneinander

**Reply by E-Stegemann** [reply]
Vielen Dank für den Hinweis! Das kann ich gut für meine Kameras übernehmen.
